q2_feature_classifier/_taxonomic_classifier.py

```diff
@@ -69,7 +69,17 @@ def _1(dirfmt: TaxonomicClassiferTemporaryPickleDirFmt) -> Pipeline:
         tmpdir = model.DirectoryFormat()
         dirname = str(tmpdir)
         tar.extractall(dirname)
-        pipeline = joblib.load(os.path.join(dirname, 'sklearn_pipeline.pkl'))
+        try:
+            pipeline = joblib.load(os.path.join(dirname,
+                                                'sklearn_pipeline.pkl'))
+        except MemoryError:
+            raise MemoryError("The operation has run out of available memory. "
+                              "To correct this error:\n"
+                              "1. Reduce the reads per batch\n"
+                              "2. Reduce number of n_jobs being performed\n"
+                              "3. Try using a smaller data set\n"
+                              "4. Use a more powerful machine or allocate "
+                              "more resources ")
```

Comment:
```suggestion
                              "more resources")
```

Reply:
Hello @thermokarst,

Regarding your comment on the `MemoryError` placement https://github.com/qiime2/q2-feature-classifier/pull/146#discussion_r314782289,

I have been searching through the forum to determine where the `MemoryError` might be occurring. If the load step is not the issue, and if the user is not specifying `n-jobs` (leads me to believe that the number of jobs is not necessarily the direct cause of the error), does that mean that the problem might be in `classifier.py` rather than the transformer?

Reply:
Looks like this message might've gotten posted in the wrong thread, I will follow up in the original thread.

Reply:
For some reason, I couldn't respond to that specific thread, so I decided to submit my question ( for the time being) in this one. Thanks!

Reply:
![image](https://user-images.githubusercontent.com/274668/63292811-643d0f80-c27b-11e9-992b-4240cb406878.png)

My PR view shows a reply box when I scroll _up_, which is weird. YMMV.